src/Workspaces/Core/Portable/Rename/ConflictEngine/ConflictResolver.Session.cs

```diff
@@ -229,6 +230,23 @@ public async Task<ConflictResolution> ResolveConflictsAsync()
 #if DEBUG
                     await DebugVerifyNoErrorsAsync(conflictResolution, _documentsIdsToBeCheckedForConflict).ConfigureAwait(false);
 #endif
+
+                    // Step 5: Rename declaration files
+                    if (_optionSet.GetOption(RenameOptions.RenameFile))
+                    {
+                        var definitionLocations = _renameLocationSet.Symbol.Locations;
+                        var definitionDocuments = definitionLocations
+                            .Select(l => conflictResolution.OldSolution.GetDocument(l.SourceTree))
+                            .Distinct()
+                            .WhereNotNull();
```

Comment:
Why the WhereNotNull? Something seems very off if that's possible.

Reply:
If only there was some way to know that GetDocument would never return null ;) 

I can remove, it's just habit because I didn't want null values. If GetDocument shouldn't return null then I'm happy to NullRef if it does